README.md

```diff
@@ -47,18 +47,18 @@ $ npm install -g bitcoin-ef
 Here's the getting started with the Extended Format
 
 ```javascript
-import { StandardToExtended, ExtendedToStandard } from '@TAAL-GmbH/bitcoin-ef';
+import { StandardToExtended, ExtendedToStandard } from '@bitcoin-ef/bsv';
```

Comment:
It should be `from 'bitcoin-ef/bsv';`

This library isn't namespaced, it will be migrated into the new BSV NPM library ecosystem coming soon under the `@bsv/...` namespace